src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.Unix.cs

```diff
@@ -41,7 +41,7 @@ private static string GetIcuLoadFailureMessage()
                 if (OperatingSystem.IsBrowser() || OperatingSystem.IsAndroid() ||
                     OperatingSystem.IsIOS() || OperatingSystem.IsTvOS() || OperatingSystem.IsWatchOS())
                 {
-                    return "Unable to load required ICU Globalization data. Please see https://aka.ms/dotnet-missing-libicu for more information";
+                    return "Unable to load required ICU Globalization data. Please see https://aka.ms/dotnet-missing-libicu for more information.\n" + Environment.StackTrace;
```

Comment:
Do we want to ship this or was it only for debugging purposes?

Reply:
I anticipate the answer is yes

Reply:
I reverted that change, it was not relevant to this PR